app/views/api/v1/lookup_keys/show.json.rabl

```diff
@@ -1,3 +1,3 @@
 object @lookup_key
 
-attributes :key, :is_param, :required, :override, :description, :default_value, :id
```

Comment:
So as not to break the API, check if you can keep this old attribute name with an appropriate value.